Core/clim-core/presentation-defs.lisp

```diff
@@ -1752,7 +1752,14 @@ protocol retrieving gestures from a provided string."))
   (defconstant +completion-options+
     '((name-key 'default-completion-name-key)
       documentation-key
-      (partial-completers '(#\Space)))))
+      (partial-completers '(#\Space))
+      (printer #'write-token)
+      (highlighter #'highlight-completion-choice))))
+
+;;;; This function is copied from CLIM Franz code
+(defun highlight-completion-choice (continuation object stream)
```

Comment:
You forgot to wrap `defun highlight-completion-choice` in an `eval-when` as this is failing to compile

Reply:
ok, that might not be it, as I still keep getting
```
; caught ERROR:
;   (during macroexpansion of (%DEFINE-PRESENTATION-TYPE COMPLETION ...))
;   The function CLIM-INTERNALS::HIGHLIGHT-COMPLETION-CHOICE is undefined.
;   It is defined earlier in the file but is not available at compile-time.
```

Reply:
ok, that was it, but I still had my flatpak build in my directory without the eval-when and was confusing asdf when I tried to build outside of flatpak. So my second comment was my fault, but my first comment still applies.

Reply:
Want me to submit a pr with the fix? I can do that if you want.

Reply:
I think you guys are doing a pretty good job developing McCLIM btw, and you guys are probably a lot better at it than I am. I'm still just learning how to use the toolkit.

Reply:
Hey, thank you for analyzing and reporting that issue. Yes, please make a pull request with a fix.